seed/challenges/05-apis-and-microservices/basic-node-and-express.json

```diff
@@ -149,13 +149,14 @@
-        "We are going to use one environment variable as a configuration option. Store the variable MESSAGE_STYLE=uppercase in the .env file. Then change the GET /json route handler so that if process.env.MESSAGE_STYLE equals to \"uppercase\" the response object becomes {message: HELLO JSON}."
+        "The .env file is an hidden file that is used to pass environment variables to your application. This file is secret, no one but you can access it, and it can be used to store data that you want to keep private or hidden. For example, you can store API keys from external services or your database URI. You can also use it to store configuration options. By setting configuration options, you can change the behavior of your application, without the need to rewrite some code. ",
+        "The environment variables are accessible from the app as <code>process.env.VAR_NAME</code>. The <code>process.env</code> object is a global Node object, and variables are passed as strings. By convention, the variable names are all uppercase, with words separated by an underscore. The .env is a shell file, so you don’t need to wrap names or values in quotes. It is also important to note that there cannot be space around the equals sign when you are assigning values to your variables, e.g. <code>VAR_NAME=value</code>. Usually, you will put each variable definition on a separate line.",
+        "Let’s add an environment variable as a configuration option. Store the variable <code>MESSAGE_STYLE=uppercase</code> in the .env file. Then tell the GET <code>/json</code> route handler that you created in the last challenge to transform the response object’s message to uppercase if <code>process.env.MESSAGE_STYLE</code> equals <code>uppercase</code>. The response object should become <code>{“message”: “HELLO JSON”}</code>."
       ],
       "challengeSeed": [],
       "tests": [
         {
-          "text" : "The response of the endpoint /json should change setting the environment variable MESSAGE_STYLE",
+          "text" : "The response of the endpoint /json should change according to the environment variable <code>MESSAGE_STYLE</code>",
```

Comment:
@erictleung cool - I'll get those in tonight. This could be done for all the backend challenges, but if I keep submitting PRs I'll never actually get through them :smile: